clang/test/CheckedCRewriter/addrof_crash.c

```diff
@@ -0,0 +1,47 @@
+// RUN: cconv-standalone %s -- | FileCheck %s
+// RUN: cconv-standalone %s -- | %clang_cc1 -fignore-checkedc-pointers -fno-builtin -verify -fcheckedc-extension -x c -
+// expected-no-diagnostics
+
+// No conversions expected for these two, they just shouldn't crash
+
+extern void foo(int* a : itype(_Array_ptr<int>));
```

Comment:
If we are going to have `Array_ptr` types in the test, we should be running with `-alltypes`. It's good that this works without it, but maybe confirm that `-alltypes` doesn't mess things up?